### DISCUSSION TITLE
error: unrecognized arguments: --rtmpdump

### DISCUSSION THREAD

**Discussion by Hveodrungr** [streamlink/streamlink#5220] [Q&A] [answered]
### Description

Hi,

I didn't use streamlink in a long time, so I updated it just in case and since the update I keep getting this error

`streamlink: error: unrecognized arguments: --rtmpdump=C:\Program Files\Streamlink\rtmpdump\rtmpdump.exe`

I checked if I had the executable, which I didn't, so I downloaded it from rtmpdump.mplayerhq.hu but it didn't change anything the error is still there.

I use streamlink 5.3.1 and downloaded rtmpdump 2.3

**Comment by mkbloke** [comment, marked as answer]
https://github.com/streamlink/streamlink/releases/tag/3.0.0
